cmd/serve.go

```diff
@@ -50,6 +52,13 @@ func (s *ServerService) StartMetricsServe(opts serve.MetricsServeOptions, httpSe
 	}
 }
 
+func (s *ServerService) StartTenantServe(opts serveTenants.ServeOptions, httpServer serveTenants.HTTPServerInterface) {
```

Comment:
I think it's more descriptive to call this `Admin` instead of `Tenant`. 
e.g. `StartAdminServe`, `admin-serve-port` etc. 

The main reason is this is going to be a server to admin the SDPs.

Reply:
Changed: [216813b](https://github.com/stellar/stellar-disbursement-platform-backend/pull/90/commits/216813b7e3d4c3fa274c821507e124c186ac8e7a) and [ad5acda](https://github.com/stellar/stellar-disbursement-platform-backend/pull/90/commits/ad5acda56559b6bf2f2457fda81c343544849501)